src/client/actions/GuildDelete.js

```diff
@@ -15,7 +15,7 @@ class GuildDeleteAction extends Action {
     let guild = client.guilds.cache.get(data.id);
     if (guild) {
       for (const channel of guild.channels.cache.values()) {
-        if (channel.type === 'text') channel.stopTyping(true);
+        if (channel.type === 'GUILD_TEXT') channel.stopTyping(true);
```

Comment:
This should probably be changed to include all text based channels, `TextBasedChannelTypes.includes(channel.type)` I believe